source/rules/no-unsafe-takeuntil.ts

```diff
@@ -83,7 +84,7 @@ const rule = ruleCreator({
     const { couldBeObservable } = getTypeServices(context);
 
     return {
-      [`CallExpression[callee.property.name='pipe'] > CallExpression[callee.name=${checkedOperatorsRegExp}]`]:
+      [`CallExpression[callee.property.name='pipe'] > CallExpression[callee.name=${checkedOperatorsRegExp}], CallExpression[callee.property.name=${checkedOperatorsRegExp}]`]:
```

Comment:
This will match calls via member expressions that are outside of `pipe`, as the `,` separates the member-expression selector from the `pipe` selector.

If you're curious, you can verify that this is how the selectors behave using http://estools.github.io/esquery/ with:

```
of(null).pipe(foo())
x.bar()
```

as the source and

```
CallExpression[callee.property.name='pipe'] > CallExpression[callee.name='foo'], CallExpression[callee.property.name='bar']
```

as the selector.

I'm going to merge this and fix it with a subsequent commit. My preference, in these situations, is to have two separate selectors/properties calling shared function.